ns2-stat-api/src/main.rs

```diff
@@ -0,0 +1,53 @@
+use actix_web::{get, web::Data, App, HttpServer, Responder};
+use clap::Parser;
+use ns2_stat::{types::GameStats, Games, NS2Stats};
+use serde::{Deserialize, Serialize};
+use std::{fs, io, path::PathBuf};
+
+struct AppData {
+    games: Vec<GameStats>,
+    newest: u32,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct DatedData<T> {
+    date: u32,
+    data: T,
+}
+
+#[get("/players")]
+async fn get_players(data: Data<AppData>) -> impl Responder {
+    serde_json::to_string(&DatedData {
+        date: data.newest,
+        data: NS2Stats::compute(Games(data.games.iter()).filter_genuine_games()),
```

Comment:
I'd precompute the `NS2Stats`, since that will likely be the most requested data.